x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -294,14 +295,17 @@ export class VectorStyle extends AbstractStyle {
     return this._isOnlySingleFeatureType(VECTOR_SHAPE_TYPES.POLYGON);
   };
 
-  _getFieldMeta = fieldName => {
-    const fieldMetaFromLocalFeatures = _.get(this._descriptor, ['__styleMeta', fieldName]);
-
+  _getDynamicPropertyByFieldName = fieldName => {
```

Comment:
Why could this not be an actual method on the prototype?

`_getDynamicPropertyByFieldName`

writing this as instance-methods with lambdas is confusing, it communicates the intent that this function is going to be passed around by reference (e.g. as a callback). Instance-methods also get recreated with every `new` call